### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by tunnelview**
Hi @StevenM42, 

please check for formatting, naming and logic. 

Thanks much :)

Shibin

### CODE REVIEW

File: prac_05/emails.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+EMAIL_ADDRESS_TO_NAME = {"shibin.abraham@gmail.com": "Shibin Abraham", "digitallearning@gmail.com": "Andrew",
```
Comment (lindsaymarkward) [inline prac_05/emails.py:2]: If this is a CONSTANT, then you should not change it. We wanted a normal variable for this one.

File: prac_05/word_occurrences.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+count_of_words = dict()  # Step 2 - Convert to Dictionary
+words = user_words.split()  # Step 3 - Splitting string into 'words', storing in 'words'
+
+for every_word in words:  # Step 4 - Count the frequency of words in "words"
+    if every_word in count_of_words:
+        count_of_words[every_word] += 1  # If the words is coming twice, iterate
+    else:
+        count_of_words[every_word] = 1
+data_of_words = []  # Step 5 - declare empty list to store 2 things - Words - & their respective frequencies
+for word_name, word_frequency in count_of_words.items():
```
Comment (lindsaymarkward) [inline prac_05/word_occurrences.py:13]: This doesn't seem helpful. Why iterate through items and then not use word_frequency? You can just iterate through the keys if you don't need the value. Plus, this just gets you a list of the keys, which you can do in one statement (see solutions).

Comment (lindsaymarkward) [reply prac_05/word_occurrences.py:13]: I just realised I'm commenting on prac 5 work in your prac 8 pull request. See if you can learn how to create PRs/branches that only contain new/desired work.

File: prac_08/silver_service_taxi.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from prac_08.taxi import Taxi
+
+
+class SilverServiceTaxi(Taxi):
+    flagfall = 4.50
+
+    def __init__(self, name, fuel, fanciness):
+        super().__init__(name, fuel)
+        self.fanciness = fanciness * self.price_per_km
```
Comment (lindsaymarkward) [inline prac_08/silver_service_taxi.py:9]: This should set the price, not change the fanciness.

File: prac_08/silver_service_taxi_test.py [hunk 4]
```diff
@@ -0,0 +1,11 @@
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    silver_service_taxi = SilverServiceTaxi("Hummer", 100, 2)
+    silver_service_taxi.drive(18)
+    print(silver_service_taxi)
+    print(silver_service_taxi.get_fare())
```
Comment (lindsaymarkward) [inline prac_08/silver_service_taxi_test.py:8]: How do we know if this is right?

File: prac_08/unreliable_car_test.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from prac_08.unreliable_car import UnreliableCar
+
+
+def main():
+    unreliable_car = UnreliableCar("Prius 1", 100, 56)
+    unreliable_car.drive(40)
+    print(unreliable_car)
```
Comment (lindsaymarkward) [inline prac_08/unreliable_car_test.py:7]: Multiple tests are helpful when dealing with random situations. Consider using a loop.

Comment (tunnelview) [reply prac_08/unreliable_car_test.py:7]: @lindsaymarkward Thanks for the feedback.

File: prac_05/emails.py (hunk 1 above)
Comment (tunnelview) [reply prac_05/emails.py:2]: Thanks much.

File: prac_05/word_occurrences.py (hunk 2 above)
Comment (tunnelview) [reply prac_05/word_occurrences.py:13]: Sure Lindsay. Thanks much.

**Pull request merged**